stellar/processors.js

```diff
@@ -51,7 +51,13 @@ const deploy = async (options, config, chain, contractName) => {
         initializeArgs: serializedArgs,
     };
 
-    printInfo(contractName, JSON.stringify(chain.contracts[contractName], null, 2));
+    printInfo('✅ Contract deployed successfully', {
```

Comment:
```suggestion
    printInfo('Contract deployed successfully', {
```
print info adds colour for you. we can make it do other things too, like add a check mark if a certain arg was set instead of having to add it manually ourselves